### DISCUSSION TITLE
wazuh agent reports not showing on dashbord

### DISCUSSION THREAD

**Discussion by ambadasjagtap2021** [wazuh/wazuh#13619] [Q&A] [answered]
wazuh agent reports not showing on dashboard.
![Untitled](https://user-images.githubusercontent.com/102735215/171011645-4870cf66-c56f-4641-b006-dfd728ea7bde.png)
![Untitled](https://user-images.githubusercontent.com/102735215/171011754-dda33d26-ecc2-4284-96c2-39bb07b5f5f2.png)

**Comment by ambadasjagtap2021** [comment]
I have installed wazuh 4.3 latest version

**Comment by noxit0** [comment, marked as answer]
Hello team!

Thank you for attaching the images.

I ask you, by any chance you have selected a time range when filtering the events? 
Maybe the time and day filtering is affecting the visualization.

Does this happen with all agents? 
In the Security Events module, do you have events registered?

I'll be waiting for your response.

Regards.
